src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -306,14 +307,17 @@ function generateComposeFiles(dockerfileName: string, platform: Platform, os: Pl
     // Ensure the path scaffolded in the Dockerfile uses POSIX separators (which work on both Linux and Windows).
     dockerfileName = dockerfileName.replace(/\\/g, '/');
 
-    let composeFileContent = dotNetComposeTemplate.replace('$service_name$', serviceName)
-        .replace(/\$image_name\$/g, imageName)
+    const normalizedServiceName = getValidImageName(serviceName);
```

Comment:
Line 279 has toLower() which is not needed anymore.
Also you can simply reuse the normalizedServiceName for the normalizedImagename

Reply:
Oh I see, I was going to ask you if they're any different. Thanks, I'll do that change!